.github/labeler.yml

```
@@ -2,19 +1,20 @@
+"Type: Docs":
 - changed-files:
   - any-glob-to-any-file: ['docs/*', 'guides/*', '**/*.md']
 
-"PR: CI":
+"Type: CI":
 - changed-files:
   - any-glob-to-any-file: ['.github/*', '**/*.yml', '.gitattributes', '.gitignore']
 
-"PR: JSON":
+"JSON":
 - changed-files:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)